# Automatically generated by Pynguin.
import semantic_release.helpers as module_0
import urllib3.util.retry as module_1

def test_case_0():
    try:
        int_0 = 1097
        bool_0 = False
        float_0 = 2640.2
        list_0 = [int_0]
        logged_function_0 = module_0.LoggedFunction(float_0)
        tuple_0 = ()
        retry_0 = module_1.Retry(list_0, logged_function_0, tuple_0)
        tuple_1 = (float_0, retry_0)
        logged_function_1 = module_0.LoggedFunction(tuple_1)
        var_0 = logged_function_1.__call__(bool_0)
        session_0 = module_0.build_requests_session(int_0, int_0)
        str_0 = None
        logged_function_2 = module_0.LoggedFunction(str_0)
        bool_1 = False
        dict_0 = {bool_1: str_0}
        var_1 = logged_function_2.__call__(dict_0)
        session_1 = module_0.build_requests_session(bool_1)
        logged_function_3 = module_0.LoggedFunction(bool_1)
        float_1 = -52.10577
        list_1 = [bool_1, session_1, float_1]
        session_2 = module_0.build_requests_session(list_1, bool_1)
        session_3 = module_0.build_requests_session()
        logged_function_4 = module_0.LoggedFunction(session_3)
    except BaseException:
        pass